### PR TITLE
Feature/download as pdf

### PR DESCRIPTION
**Pull request by LewisDaleUK**

### CODE REVIEW

File: judgments/views.py [hunk 1]
```diff
@@ -111,6 +115,27 @@ def detail_xml(_request, judgment_uri):
...
+        return super(PdfDetailView, self).dispatch(request, *args, **kwargs)
+
+    def get_context_data(self, judgment_uri, **kwargs):
+        context = super().get_context_data(**kwargs)
+
+        results = api_client.eval_xslt(judgment_uri)
+        multipart_data = decoder.MultipartDecoder.from_response(results)
+        context["judgment"] = multipart_data.parts[0].text
+
+        print(context["judgment"])
```
Comment (lozette) [inline judgments/views.py:135]: Remove me!

File: ds_judgements_public_ui/templates/includes/judgment_text_toolbar.html [hunk 2]
```diff
@@ -8,6 +8,7 @@
         </div>
         <div class="judgment-toolbar__download">
             <a class="judgment-download__option--xml" href="{% url 'detail_xml' context.judgment_uri %}">{% translate "judgment.downloadasxml" %}</a>
+            <a class="judgment-download__option--xml" href="{% url 'detail_pdf' context.judgment_uri %}">{% translate "judgment.downloadaspdf" %}</a>
```
Comment (lozette) [inline ds_judgements_public_ui/templates/includes/judgment_text_toolbar.html:11]: I think this class should end with `--pdf` and not `--xml`

File: package.json [hunk 3]
```diff
@@ -7,8 +7,8 @@
     "doc": "docs"
   },
   "scripts": {
-    "start-sass": "sass --watch ds_judgements_public_ui/sass/main.scss:ds_judgements_public_ui/static/css/main.css",
-    "start-scripts": "npx webpack --watch"
+    "start-sass": "sass --watch ds_judgements_public_ui/sass:ds_judgements_public_ui/static/css",
+    "start-scripts": "npx webpack --watch",
```
Comment (gtvj) [inline package.json:11]: The trailing comma here is causing the `package.json` to throw a parsing error.

**Comment by gtvj** [comment]
Hi @LewisDaleUK - the front end stuff here seems fine to me. I wouldn't usually use absolute units in CSS but since this is for the PDF output only I've found in other projects where we've done this that it's fine.

**Review by lozette — APPROVED** [review]

**Pull request merged**